.github/workflows/auto-review-bot.yml

```diff
@@ -0,0 +1,60 @@
+on:
+  workflow_run:
+    workflows:
+      - Auto Review Bot Trigger
+    types:
+      - completed
+
+name: Auto Review Bot
+jobs:
+  auto-review-bot:
+    runs-on: ubuntu-latest
+    name: Run
+    steps:
+      - name: Fetch PR Number
+        uses: dawidd6/action-download-artifact@6765a42d86407a3d532749069ac03705ad82ebc6
+        with:
+          name: pr-number
+          workflow: auto-review-trigger.yml
+          run_id: ${{ github.event.workflow_run.id }}
+
+      - name: Save PR Number
+        id: save-pr-number
+        run: echo "::set-output name=pr::$(cat pr-number.txt)"
+
+      - name: Checkout
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+        with:
+          repository: ethereum/EIPs # Default, but best to be explicit here
+          ref: master
+
+      - name: Setup Node.js Environment
+        uses: actions/setup-node@2fddd8803e2f5c9604345a0b591c3020ee971a93
+        with:
+          node-version: 14
+
+      - name: Auto Review Bot
+        id: auto-review-bot
+        uses: Pandapip1/EIP-Bot@patch-5
+        with:
+          GITHUB-TOKEN: ${{ secrets.TOKEN }}
+          PR_NUMBER: ${{ steps.save-pr-number.outputs.pr }}
+          CORE_EDITORS: '@MicahZoltu,@lightclient,@axic,@gcolvin,@SamWilsn,@Pandapip1'
+          ERC_EDITORS: '@lightclient,@axic,@SamWilsn,@Pandapip1'
+          NETWORKING_EDITORS: '@MicahZoltu,@lightclient,@axic,@SamWilsn'
+          INTERFACE_EDITORS: '@lightclient,@axic,@SamWilsn,@Pandapip1'
+          META_EDITORS: '@lightclient,@axic,@gcolvin,@SamWilsn,@Pandapip1'
+          INFORMATIONAL_EDITORS: '@lightclient,@axic,@gcolvin,@SamWilsn,@Pandapip1'
+          MAINTAINERS: '@alita-moore,@mryalamanchi'
```

Comment:
Do we need to update [EIP-5069](https://eips.ethereum.org/EIPS/eip-5069)?

Reply:
EIP-Bot is out of date there, but that's outside of the scope of this PR.

Reply:
Oh, I misunderstood you. Yes, I'll update it.